packages/nx-plugin/src/plugin/target/configuration-target.ts

```diff
@@ -1,12 +1,22 @@
 import { TargetConfiguration } from '@nx/devkit';
 import { RunCommandsOptions } from 'nx/src/executors/run-commands/run-commands.impl';
 import { PACKAGE_NAME } from '../../internal/constants';
+import { CP_TARGET_NAME } from '../constants';
 
-export function createConfigurationTarget(
-  projectName?: string,
-): TargetConfiguration<RunCommandsOptions> {
+export function createConfigurationTarget(options?: {
+  targetName?: string;
+  projectName?: string;
+  bin?: string;
+}): TargetConfiguration<RunCommandsOptions> {
+  const {
+    projectName,
+    bin = PACKAGE_NAME,
+    targetName = CP_TARGET_NAME,
+  } = options ?? {};
   const projectFlag = projectName && ` --project=${projectName}`;
   return {
-    command: `nx g ${PACKAGE_NAME}:configuration${projectFlag ?? ''}`,
+    command: `nx g ${bin}:configuration --skipTarget --targetName=${targetName}${
```

Comment:
Curious what the reason is for adding `--skipTarget`? :thinking:

Reply:
To give the user the option to only create the config file. 

Internally it is used for the derived targets of the plugin. If plugin is used no need to create a target.